### PR TITLE
Updated copy for #81

### PR DESCRIPTION
**Pull request by ALRubinger**
Still TODO: add images of runtimes, Erik to review my frontend CSS and HTML crap

@edewit Mind a review and adding the images of the runtimes, centered, as indicated?  It's OK to put 'em on white BGs; I know we already have transparent images so just use your best judgement.

### CODE REVIEW

**Comment by ALRubinger** [comment]
My PR should look a little like this:
screenshot4

File: src/app/wizard/intro/intro.component.html [hunk 1]
```diff
@@ -5,28 +5,22 @@
...
-      continuous delivery, and ensure it's triggered to run on each push to your git remote.
-    </p>
-    <p>
-      Let's get you into orbit.
+      Images here: Eclipse Vert.x | WildFly Swarm | Spring Boot
     </p>
   </div>
+  <h4></h4>
+  <div class="blank-slate-pf-main-action">
+    <button class="btn btn-primary btn-lg btn-reg" (click)="launch()">
```
Comment (edewit) [inline src/app/wizard/intro/intro.component.html]: I guess we don't want to start the wizard on this button, but got to "https://manage.openshift.com/openshiftio" so better would be to make it a link:
```html
<a href="https://manage.openshift.com/openshiftio" class="btn btn-primary btn-lg btn-reg">
```

Comment (ALRubinger) [inline src/app/wizard/intro/intro.component.html]: +1.  This is also described in #83

**Comment by ALRubinger** [comment]
Ready!

**Comment by ALRubinger** [comment]
screenshot12

**Pull request merged**